native/acter/Cargo.toml

```diff
@@ -40,8 +39,8 @@ mime = "0.3.17"
 mime2ext = "0.1.52"
 mime_guess = "2.0.4"
 parse-env-filter = "0.1.0"
-ruma = { workspace = true, features = ["html", "unstable-unspecified", "unstable-msc3488", "compat-unset-avatar", "unstable-msc3245-v1-compat"] }
-ruma-common = { workspace = true, features = ["client"] }
+ruma = { workspace = true, features = ["rand", "unstable-msc2448", "unstable-msc2965", "unstable-msc3930", "unstable-msc3245-v1-compat"] }
```

Comment:
`html` disappeared. is that on purpose?

Reply:
Ah, I cloned params from matrix-rust-sdk project.
Will revert `html` feature.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1311/commits/099616d7834d0a61c7c2d8e83dffd34a27b71d9b